docs/spec-subscriptions.md

```diff
@@ -1,7 +1,21 @@
 ---
 id: spec-subscriptions
-title: spec-subscriptions
-sidebar_label: spec-subscriptions
+title: Subscriptions
 ---
 
-spec-subscriptions TODO
+Subscriptions are divided to 3 different groups of changes
+Create, Update and Delete subscriptions that users can use with filtering capabilities
+
+``` graphql
+input NoteSubscriptionFilter {
+  id: ID
+  title: String
```

Comment:
Why is the type `String` and not `StringInput` ?

Reply:
That is subscription filtering. For subscriptions, we do not support advanced filters at the moment. You can filter only by single field. Good point this needws to be explained

Reply:
Great.